Bottom: aa49feb28d407f56f56cbaecd45834d9fa2eb154
Top:    fcb5a3ae2ac64db490c76b877583c238e99d45ed
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-09 07:46:22 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
